composer.json

```diff
@@ -22,11 +22,11 @@
         "php": "^7.1 || ^8.0",
         "php-http/promise": "^1.1",
         "psr/http-client": "^1.0",
-        "psr/http-message": "^1.0"
+        "psr/http-message": "^1.0 || ^2.0"
```

Comment:
are the test failures related to installing with 2.0? the lowest builds are all green.

Reply:
fixed, all green!